simulator/src/embed.js

```diff
@@ -56,6 +56,12 @@ function toggleFullScreen(value) {
 function exitHandler() {
     setTimeout(() => {
         Object.keys(scopeList).forEach((id) => {
+            if(document.getElementById("fullscreen").value == "Full Screen") {
```

Comment:
Does this have to happen for each element in the scopeList?

Reply:
yes sir